lib/pbench/test/unit/server/conftest.py

```diff
@@ -121,8 +123,8 @@ def server_config(on_disk_server_config, monkeypatch) -> PbenchServerConfig:
     return server_config
 
 
-@pytest.fixture
-def client(server_config):
+@pytest.fixture()
+def client(server_config, fake_email_validator):
```

Comment:
Yeah, these are possibly excessive; I was worried about whether the Flask client setup was causing the trouble, and added these during my cautious scientific investigations (or "frenzied thrashing" ... take your pick). I'll try removing them.

Reply:
And ... `pbench_admin_token` should be (and is) redundant because it's using the Flask client set up by the `client` fixture. The latter, however, appears to be necessary; to enable the `register_user` helper; it's quite possible (even likely) that following the use of that to actual test cases and annotating them for the fixture would be sufficient. There are, however, many, scattered across various tests; enough that I think annotating it once here makes sense. But I'll add a comment.